.github/workflows/build.yml

```diff
@@ -6,12 +6,6 @@ on:
 
 env:
   CMAKEFLAGS_COMMON: -DCMAKE_BUILD_TYPE=Release -DBATTERY=ON -DBROADCAST=ON -DBULK=ON -DDEBUG_ASSERTIONS_FATAL=ON -DHID=ON -DLILV=ON -DOPUS=ON -DQTKEYCHAIN=ON -DVINYLCONTROL=ON
-  GTEST_COLOR: 1
-  CTEST_OUTPUT_ON_FAILURE: 1
-  # Render analyzer waveform tests to an offscreen buffer
-  QT_QPA_PLATFORM: offscreen
-  # GitHub Actions automatically zstd compresses caches
-  CCACHE_NOCOMPRESS: true
```

Comment:
CCACHE_NOCOMPRESS got lost

Reply:
Oh, I see you set it below. Nevertheless, whenever Ubuntu updates to CMake 4.0 this should also be set for Ubuntu so I think it would be good to keep it in this global `env` section.

Reply:
What do you mean? It's set for the build step which is used by both macOS and Ubuntu.

Reply:
Oh, nevermind then.